nimble/host/mesh/src/proxy_srv.c

```diff
@@ -429,7 +431,7 @@ static int node_id_adv(struct bt_mesh_subnet *sub, int32_t duration)
 	memcpy(proxy_svc_data + 3, tmp + 8, 8);
 
 	err = bt_mesh_adv_start(&fast_adv_param, duration, node_id_ad,
-			      ARRAY_SIZE(node_id_ad), sd, 0);
+                            ARRAY_SIZE(node_id_ad), sd, sd_len);
```

Comment:
couldn't this just accept NULL instead?

Reply:
Now that I look at it, I see that it just passes `sd` into `bt_le_adv_start` - so I think yes, it already accepts NULL and this is false positive